helper_functions.py

```diff
@@ -296,4 +296,39 @@ def fit_power_law_alpha(z,U,zref=80.0,Uref=8.0):
     R2 = 1.0 - (SSres/SStot)
     return alpha, R2
 
+def model4D_calcQOIs(ds,mean_dim):
+    """
+    Augment an a2e-mmc standard, xarrays-based, data structure of 
+    4-dimensional model output with space-based quantities of interest
+
+    Usage
+    ====
+    ds : mmc-4D standard xarray DataSet 
+        The raw standard mmc-4D data structure 
+    mean_dim : string 
+        Dimension along which to calculate mean and fluctuating (perturbation) parts
+    """
 
+    dim_keys = [*ds.dims.keys()]
+    ds_means = ds.mean(dim=mean_dim)
```

Comment:
This presumes that the data are stationary and/or homogeneous? Perhaps worth noting in the docstring.